wallaby.js

```diff
@@ -1,14 +1,23 @@
-module.exports = function (wallaby) {
+module.exports = function(wallaby) {
     return {
+        env: {
+            type: 'node',
+            runner: 'node',
+        },
         files: [
-            {pattern: 'scripts/**/*.js'},
+            'scripts/**/*.js',
```

Comment:
I shortened this to just the pattern string. You can mix and match each one. Typically I use the object form (with a pattern property) only if I need some additional configuration such as `instrument: true|false`, etc.